views/partials/header.ejs

```diff
@@ -1,6 +1,44 @@
-<div id='header'>
-    <p class ='headerItem' onclick="location.href='../candidates/new'">New candidate</p>
-    <p class ='headerItem' onclick="location.href='../candidates'">All candidates</p>
-    <p class ='headerItem' onclick="location.href='../jobs/new'">New Job</p>
-    <p class ='headerItem' onclick="location.href='../jobs'">All Jobs</p>
-</div>
\ No newline at end of file
+<header>
+
+  <nav class="navbar navbar-expand-xl navbar-dark">
+    <a class="navbar-brand headerItem mx-lg-5" href="/">ReMatch</a>
+
+    <%# button to toggle menu on mobile screen %>
+    <button class="navbar-toggler ml-auto mr-lg-3" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <ul class="navbar-nav ml-auto">
+      <%# links when logged in %>
```

Comment:
Shouldnt FE get this message from BE first (for us with Josh) and then be merged to main?

Reply:
Can not test it on work device. But if you need this flash messages  first, better give us a warning to not to merge it to main branch.